interfaces/builtin/helpers.go

```diff
@@ -80,7 +80,7 @@ func addLdconfigLibDirs(spec *ldconfig.Specification, slot *interfaces.Connected
 // dir, for different instances and slots.
 func librarySourcePath(instance, slotName, ifaceName string) string {
```

Comment:
maybe we want to add system to the front of this? same for the helpers using this?

Reply:
Done